### DISCUSSION TITLE
求助贴

### DISCUSSION THREAD

**Discussion by fu-fu404** [share121/inter-knot#1267] [其他]
绳网的消息邮箱一直收到怎么办啊，有大佬能帮忙的吗？我出1丁尼

**Comment by lllliufeng** [comment]
- [ ] Notifications
- [ ] Manage notifications
- [ ] Notification settings
- [ ] Subscriptions - Participating, @mentions and custom 
- [ ] Notify me on - Email勾掉

**Comment by Buer-Nahida** [comment]
见https://github.com/share121/inter-knot/discussions/512
